cellstore/reader.go

```diff
@@ -176,6 +178,84 @@ func (r *Reader) readBlock(pos int) ([]byte, error) {
 	}
 }
 
+type Near struct {
+	CellID   s2.CellID
+	Value    []byte
+	Distance int32
+}
+
+type Nearby []*Near
```

Comment:
I don't think you need two slice types. plus, I would prefer to keep them private and as non-pointers